.buildkite/pipeline.yml

```
@@ -1,5 +1,7 @@
 env:
+  SLURM_CPUS_PER_TASK: 1
   OPENBLAS_NUM_THREADS: 1
+  CLIMATEMACHINE_SETTINGS_FIX_RNG_SEED: "true"
```

```suggestion
  OPENBLAS_NUM_THREADS: 1
```
Slurm should use 1 CPU per task by default. I also think that `CLIMATEMACHINE` flag won't work as intended.